src/libraries/RegistryOps.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.21;
+
+library RegistryOps {
+    /// @dev `x > y ? x : y`.
+    function max(uint256 x, uint256 y) internal pure returns (uint256 z) {
+        assembly ("memory-safe") {
+            z := xor(x, mul(xor(x, y), gt(y, x)))
```

Comment:
This is a great solution from what I can see. It would be worth adding some comments as to why this works for all inputs in relation to underflow / overflow possibility with the mul (which, from what I can see, won't happen :) ) and maybe explain why it works for devs that might use it.